src/garage/envs/base.py

```diff
@@ -148,7 +148,19 @@ def step(self, action):
                 debugging, and sometimes learning)
 
         """
-        return self.env.step(action)
+        observation, reward, done, info = self.env.step(action)
+        # gym envs that are wrapped in timelimit wrapper modify
+        # the done/termination signal to be true whenever a time
+        # limit expiration occurs. The following statement sets
+        # the done signal to be True only if caused by an
+        # environment termination, and not a time limit
+        # termination. The time limit termination signal
+        # will be saved inside env_infos as
+        # 'TimeLimitTerminated'
```

Comment:
can you make this `GarageEnv.Terminated`? It will help downstream users figure out where the key came from.

Reply:
perhaps the more useful signal would be `GarageEnv.terminal_corrected`

Reply:
`GarageEnv.terminal_corrected` implies we are passing the corrected terminal value in the dict. Aren't we already doing this with the Done value?

I will change the signal name from `TimeLimitTerminated` to `GarageEnv.Terminated`

Reply:
`TimeLimit.truncated` is already in `info` and can tell the user what the original value was. I wanted to provide a signal in `info` that done has been modified.

Reply:
if you simply want to store the original value (as here) then i suggest you put it at `GarageEnv.TimeLimit.done` and also add `GarageEnv.done_corrected = True`